src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
Divide logic, create separate methods
```
readFromFile
generateReport / calculateReport
writeIntoFile
```

Create constant for supply, buy, result and comma symbol
Use these constants. In report case: append(SUPPLY).append(COMMA).append(.....